src/hooks/useCurrentMonth.ts

```diff
@@ -0,0 +1,42 @@
+// hook that returns information about the current month
+export const useCurrentMonth = () => {
+  const today = new Date();
+  const month = today.toLocaleString("default", { month: "long" });
+  const year = today.getFullYear();
+
+  // returns the shorthand weekday name for a given Date
+  function getDayOfTheWeek(day: Date): string {
+    return day.getDay() === 4
+      ? "R"
```

Comment:
Yes, "R" represents Thursday. And yes, "Th" makes more sense to me, too.
I used "R" cause that's how the design looks.
I'll change it to "Th" since I agree it's easier to understand

Reply:
sure.. we can talk about it at our next meeting.